pkg/registry/storage/csidriver/strategy.go

```diff
@@ -85,8 +82,7 @@ func (csiDriverStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.
 		!utilfeature.DefaultFeatureGate.Enabled(features.CSIInlineVolume) {
 		newCSIDriver.Spec.VolumeLifecycleModes = nil
 	}
-	if oldCSIDriver.Spec.FSGroupPolicy == nil &&
-		!utilfeature.DefaultFeatureGate.Enabled(features.CSIVolumeFSGroupPolicy) {
+	if oldCSIDriver.Spec.FSGroupPolicy == nil {
 		newCSIDriver.Spec.FSGroupPolicy = nil
 	}
```

Comment:
This looks wrong. Now that features.CSIVolumeFSGroupPolicy is always enabled, the if branch should never get taken and thus needs to be removed.

Reply:
Thanks! Done.